ldm/simplet2i.py

```diff
@@ -294,24 +311,25 @@ def process_image(image,seed):
         try:
             if init_img:
                 assert os.path.exists(init_img), f'{init_img}: File not found'
-                images_iterator = self._img2img(
+                init_image = self._load_img(init_img, width, height, fit).to(self.device)
+                with scope(device.type):
```

Comment:
I believe this needs to be self.device.type. Fixing in preparation for merge.

Reply:
Ah yeah, I must have messed that up in one of the rebases.